src/compiler/moduleNameResolver/moduleNameResolver.ts

```diff
@@ -2883,7 +2899,7 @@ function tryLoadModuleUsingPaths(extensions: Extensions, moduleName: string, bas
             trace(state.host, Diagnostics.Module_name_0_matched_pattern_1, moduleName, matchedPatternText);
         }
         const resolved = forEach(paths[matchedPatternText], subst => {
-            const path = matchedStar ? subst.replace("*", matchedStar) : subst;
+            const path = matchedStar ? subst.replace(/\*/g, matchedStar) : subst;

```

Comment:
I know it fails CI, but can you revert these changes? If these are actual bugs, we probably want to fix them elsewhere rather than sneaking them into this PR (which I hope turns into just code moves).

Reply:
Ok, reverted